tools/copy_production_db_to_staging

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${OAR_DEBUG}" ]]; then
+    set -x
+fi
+
+# Constants
+STAGING_ID="openapparelregistry-enc-stg"
+PRODUCTION_ID="openapparelregistry-enc-prd"
+ALLOWED_EMAILS="person@example.com other@example.com"
+S3_BUCKET="openapparelregistry-development-data-eu-west-1"
```

Comment:
I had written this copy to s3 into the requirements based on a previous workflow and now realize that it is not needed. To avoid piling up dumps in the bucket I recommend removing this variable.